### PR TITLE
Add Blink method to textinput to return blink state

### PR DESCRIPTION
**Pull request by aschey**
As discussed on Slack earlier, this PR adds a `Blink()` method that returns the text input's blink state. This is useful if you want to render something more complex than the existing `View()` method can handle. For context, I'm implementing an input with multiple types of placeholders, each with different styles. I thought about adding something to the text input itself to handle this case, but it would add a lot of complexity for a rather niche use case. With the `blink` property exposed, users can implement their own `View()` method.

### CODE REVIEW

File: textinput/textinput.go [hunk 1]
```diff
@@ -203,6 +203,11 @@ func (m Model) Cursor() int {
 	return m.pos
 }
 
+// Blink returns whether the cursor should blink or not.
```
Comment (meowgorithm) [inline textinput/textinput.go:206]: I'd reword this to “Blink returns whether or not to draw the cursor,” just to disambiguate it from cursor states like `CursorStatic`, `CursorBlink` and `CursorHide`.

**Comment by meowgorithm** [comment]
One small comment above, otherwise we're good to go. Thanks for this one!

File: textinput/textinput.go (hunk 1 above)
Comment (aschey) [reply textinput/textinput.go]: Thanks, I agree that sounds much better. Fixed now.

**Review by meowgorithm — APPROVED** [review]
